app/views/user/populi_transfer/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<br>
+<%= link_to sanitize("&larr; Back to Attendance"), attendance_path(@attendance) %>
```

Comment:
Could you explain what the sanitize link entails and why it's better to use it in this circumstance?

Reply:
It helps protect the Rails application from Cross Site Scripting (XSS) attacks, validates the URL format, and prevents open redirects. This context probably doesn't warrant such security measures, but it's just a security practice I have been utilizing.